src/checkpoint1/checkpoint1.css

```diff
@@ -0,0 +1,136 @@
+body {
+  display: grid;
+  grid-template-columns: repeat(6, 1fr);
+  grid-template-rows: repeat(8, .5fr);
+  background-color: gray;
+  height: 100vh;
+}
+
+aside {
+  grid-area: 1/1/-1/2;
+  background-color: black;
+  color: white;
+  display: grid;
+  grid-template-rows: inherit;
+}
+  #dash {
+    grid-column: inherit;
+    grid-row: 2/3;
+  }
+  #ui {
+    grid-column: inherit;
+    grid-row: 3/4;
+  }
+  #cal {
+    grid-column: inherit;
+    grid-row: 4/5;
+  }
+  #mail {
+    grid-column: inherit;
+    grid-row: 5/6;
+  }
+  #comp {
+    grid-row: 6/7;
+  }
+  #widgets {
+    grid-row: 7/8;
+  }
+  #tables {
+    grid-row: 8/9;
+  }
+
+header {
+  grid-area: 1/2/2/-1;
+  background-color: fuchsia;
+  color: white;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+  #upperrighticons {
+    display: flex;
+    justify-content: space-evenly;
+    /* is the div only as big as the content? cause then this does nada & i want a lil space */
+  }
+
+main {
+  grid-area: 3/2/span1/-1;
+  display: grid;
+  grid-template-columns: inherit;
+  grid-template-rows: inherit;
+}
+
+  .maindivs {
+    border: thin grey solid;
+    color: white;
+  }
```

Comment:
text touching the edge of its container is the sign of a noob. let's pretend we're pro's until we are. add some padding. i played with `padding: 20px 20px 0;` but you can add whatever you'd like.